docs/sources/cursor/example-api-responses/sanitized/filtered-usage-events.json

```diff
@@ -0,0 +1,60 @@
+{
+  "totalUsageEventsCount" : 1250,
+  "pagination" : {
+    "numPages" : 25,
+    "currentPage" : 1,
+    "pageSize" : 50,
+    "hasNextPage" : true,
+    "hasPreviousPage" : false
+  },
+  "usageEvents" : [
+    {
+      "timestamp" : null,
```

Comment:
Pls check this -> https://github.com/Worklytics/evalengin/pull/7680#discussion_r2210739177

Reply:
thanks - done.